README.md

```diff
@@ -5,19 +5,13 @@
 - Ruby
 
 
-## Live Demo
+## How to run
 
-- [Live Demo Link](https://)
+Add this: `#!/usr/bin/env ruby`
+to the beginning of `main.rb`.
 
-
-## Available Scripts
-
-In the project directory, you can:
-
-### Run the app in the development mode
-
-
-### Launch the test runner in the interactive watch mode
```

Comment:
- [OPTIONAL] Please it will be nice to add explicit steps on how to clone and run your project locally as this will be of help to non technical users.

Reply:
There is an explanation for how to run it on the last PR.